src/hooks/useViewerCallbacks.js

```diff
@@ -1,6 +1,7 @@
 import { useEffect } from 'react';
 
 export const useViewerCallbacks = ({
+  visibleColumns,
```

Comment:
Words can not describe how much I dislike the naming of these properties. They are inconsistent with how they're actually used and impressively confusing to trace.

Reply:
I see what you mean lol

Reply:
i'd change them if it didn't mean tearing through a bunch of code